### PR TITLE
Feature/map component

### PR DESCRIPTION
**Pull request by natevenn**
@davidwparker @willklein @rpeyfuss 
### Changes
- add map component that integrates with google maps api.
- add package [google-maps-react](https://github.com/fullstackreact/google-maps-react)
- set state.location to the response of the meetup api react denver location in main component.
- pass state.location to Gmap component
### Entry
- app/components/GMap
### Background
- I chose to go with google-maps-react package b/c the code is simple and good documentation.
- the map component takes the lat and long the I get back from the meetup api and pass props to the marker component which drops a pin on the map where the meetup is.

### CODE REVIEW

File: app/components/GMap.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React, {Component} from 'react';
+import {Map, Marker, GoogleApiWrapper} from 'google-maps-react'
+
+export class GMap extends Component {
+  render() {
+    var style = {
+      width: '800px',
+      height: '300px',
+    }
```
Comment (rpeyfuss) [inline app/components/GMap.js]: @natevenn style already exists in stylesheet; should we remove this?

Comment (natevenn) [reply app/components/GMap.js]: Good catch. Yes I can remove the style object from the map component.

File: app/components/Header.js [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+	render() {
+		return (
+				<nav className="ui header">
+					<div className="react-denver-logo">
+						<h2 className="logo-text-denver">Denver</h2>
+						<img className='react-logo' src="/style/images/react-logo.png" alt="react logo" />
+						<h2 className="logo-text-js">JS</h2>
+					</div>
+				</nav>
     );
```
Comment (rpeyfuss) [inline app/components/Header.js:14]: @natevenn Indents be removed?

Comment (rpeyfuss) [inline app/components/Header.js:14]: @natevenn Indents be removed?

Comment (natevenn) [reply app/components/Header.js:14]: Yes they can. good eye

File: app/components/Meetup.js [hunk 3]
```diff
@@ -1,37 +1,13 @@
 import React from 'react';
 
 class Meetup extends React.Component {
```
Comment (rpeyfuss) [inline app/components/Meetup.js]: @natevenn To stay consistent with the other components, should we use the following for imports and class declarations:

import React, {Component} from 'react';
export default class Meetup extends Component

Comment (natevenn) [reply app/components/Meetup.js]: Yes. good call. on it.

**Pull request merged**